cpp/src/Slice/Grammar.y

```diff
@@ -100,7 +101,8 @@ slice_error(const char* s)
 %token ICE_FLOATING_POINT_LITERAL
 %token ICE_IDENT_OP
 %token ICE_KEYWORD_OP
-%token ICE_OPTIONAL_OP
+%token ICE_TAG_START
```

Comment:
Why this renaming from _OP to _START?

Reply:
Because the text ICE_TAG_START corresponds to is 'tag(', which is the start of a tag.

I suspect the name ICE_TAG_OP is leftover from the fact that we scan for the token in the same function we scan for operations, where we just match any string ending with a '(', then afterwards check if it's a keyword or an operation name.

Either way, 'tag(' isn't an operation, it's the start of a tag, so I felt the renaming made it more clear. If there's an alternate meaning or something I've missed, I can switch it back. But I couldn't find one.